app/subscriber/src/css/_colors.scss

```diff
@@ -1,11 +1,17 @@
 $action-button-color: #007af5;
 $background-color: white;
 $primary-color: #003366;
-$primary-color-rgb: 0, 51, 102;
+$primary-color-rgb: 0,
+51,
+102;
 $primary-light-color: #38598a;
-$primary-light-color-rgb: 56, 89, 138;
+$primary-light-color-rgb: 56,
```

Comment:
Don't like auto formatting outcome here...  Not going to figure out why it's so ugly atm.